src/kedro_templar/process_templates.py

```diff
@@ -0,0 +1,116 @@
+import logging
+import os
+from pathlib import Path
+from typing import Text, Dict
+import click
+from . import settings, templates
+from .core import utils
+
+log = logging.getLogger(__name__)
+
+
+@click.command()
+@click.option(
+    "-c",
+    "--config",
+    "config_path",
+    required=True,
+    type=click.Path(exists=True)
+)
+@click.option(
+    "-t",
+    "--templates",
+    "templates_dir",
+    required=False,
+    default=settings.TEMPLATES_DIR,
+    type=click.Path(exists=True)
+)
+@click.option(
+    "-o",
+    "--output",
+    "output_dir",
+    required=False,
+    default=settings.OUTPUT_DIR,
+    type=click.Path(exists=True)
+)
+@click.option(
+    "-f",
+    "--force",
+    "replace_existing",
+    required=False,
+    default=False,
+    type=bool
+)
+@click.option(
+    "-e",
+    "--export",
+    "export_path",
+    required=False,
+    type=click.Path(exists=True)
+)
+def apply(
+        config_path: Text,
+        templates_dir: Text,
+        output_dir: Text,
+        replace_existing: bool
+):
+    config = utils.load_config(config_path)
+    environment = templates.create_environment(templates_dir)
+    for template_path in environment.list_templates():
+        template = environment.get_template(template_path)
+        result = template.render(config)
+        output_path = Path(output_dir) / template_path
+        utils.save_result(result, output_path, replace_existing)
+
+
+@click.command()
+@click.option(
+    "-f",
+    "--file",
+    "config_path",
+    required=True,
+    type=str
+)
+@click.option(
+    "-o",
+    "--output",
+    "output_dir",
+    required=False,
+    default=settings.CONFIG_OUTPUT_DIR,
+    type=click.Path(exists=True)
```

Comment:
are we sure it should be `exists=True`? does it check only the directory not the file?